products/zoningtaxlots/bash/export.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+source ../../bash/utils.sh
+set_error_traps
+
+echo "Generate csvs and shapefiles ..."
+rm -rf output && mkdir -p output
+(
+    cd output
+
+    cp ../source_data_versions.csv ./
+    cp ../build_metadata.json ./
+    
+    csv_export zoningtaxlots zoningtaxlot_db &
+    csv_export qa_freq qaqc_frequency &
+    csv_export qa_bbl qaqc_bbl &
+    csv_export qa_mismatch qaqc_mismatch &
+    csv_export qa_vers_comparison qc_versioncomparison &
+    csv_export qa_null qaqc_null &
+    csv_export qa_bbldiffs qc_bbldiffs &
+
+    shp_export qa_bbldiffs MULTIPOLYGON -t_srs "EPSG:2263"
```

Comment:
Last little thing - looks like the output typically is named `qc_bbldiffs`. You can do this with `shp_export qa_bbldiffs MULTIPOLYGON -f qc_bbldiffs -t_srs "EPSG:2263"` (just telling you instead of letting you check it out because the bash function is a little funky).

We should probably clean up these filenames (make them consistent, with each other and with the dbt models) but that can wait

Reply:
@HengJiang0206 I made some modifications to the qa model file names so they are consistent with the output file names

Reply:
Ah I had wanted to leave the model names. But let's keep this. 

BUT then we need one more change - the renaming in the calls to csv_export and shp_export don't need to happen anymore. So we can just have

`csv_export qa_null &`

for the csv exports, and drop the `-f "qc_bbldiffs"` from the shp_export